### PR TITLE
[#172526356] Adds Screen to explain why CIE login is not available

### PR DESCRIPTION
**Pull request by CrisTofani**
**Short description:**
This PR adds the screen that explains why CIE is not supported

**List of changes proposed in this pull request:**
- ts/store/actions/cie.ts
- ts/store/reducers/cie.ts
- ts/sagas/cie.ts
- ts/screens/authentication/LandingScreen.tsx

### CODE REVIEW

File: ts/screens/authentication/LandingScreen.tsx [hunk 1]
```diff
@@ -120,15 +136,86 @@ class LandingScreen extends React.PureComponent<Props> {
     }
   }
 
+  private renderAndroidConditions = () => {
```
Comment (Undermaken) [inline ts/screens/authentication/LandingScreen.tsx:139]: Could it be a component included in its own file?
We should pass to it `hasCieApiLevelSupport`

Comment (CrisTofani) [reply ts/screens/authentication/LandingScreen.tsx:139]: We may create a component for the complete child of the Modal including this block maybe?
```javascript
          <React.Fragment>
            <Markdown>
              {I18n.t("authentication.landing.cie_unsupported.body")}
            </Markdown>
            {Platform.OS === "android" && this.renderAndroidConditions()}
          </React.Fragment>
```

we should pass `hasCieApiLevelSupport` and `hasCieNFCFeature` too i guess

Comment (CrisTofani) [reply ts/screens/authentication/LandingScreen.tsx]: @Undermaken 
The component is available in #05798b8

File: locales/it/cie/cieNotSupported.md [hunk 2]
```diff
@@ -0,0 +1 @@
+Per effettuare il login è necessario avere un dispositivo con tecnologia NFC o un dispositivo compatibile con l’attuale implementazione (i dispositivi iOS non sono ancora compatibili ma a breve lo diventeranno con i prossimi aggiornamenti).
```
Comment (Undermaken) [inline locales/it/cie/cieNotSupported.md:1]:
```suggestion
Per effettuare il login è necessario avere un dispositivo con tecnologia NFC e il sistema operativo con i requisiti minimi necessari (i dispositivi **iOS** non sono ancora compatibili ma a breve lo diventeranno con i prossimi aggiornamenti).
```
This description, in the way we proposed, doesn't make sense

File: ts/components/cie/CieNotSupported.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+          <View spacer={true} />
+          <Markdown>
+            {I18n.t("authentication.landing.cie_unsupported.android_desc")}
+          </Markdown>
+          <View spacer={true} extralarge={true} />
+          <List>
+            <ListItem>
+              <IconFont
+                name="io-tick-big"
+                size={16}
```
Comment (Undermaken) [inline ts/components/cie/CieNotSupported.tsx:31]: Avoid magic number
Place it as const on top of file

File: ts/screens/authentication/LandingScreen.tsx [hunk 4]
```diff
@@ -89,6 +98,12 @@ const styles = StyleSheet.create({
   },
   flex: {
     flex: 1
+  },
+  noCie: {
+    opacity: 0.35
+  },
+  checkboxBackground: {
```
Comment (Undermaken) [inline ts/screens/authentication/LandingScreen.tsx:105]: It seems to be not used

File: ts/store/reducers/cie.ts [hunk 5]
```diff
@@ -38,6 +44,26 @@ export default function cieReducer(
...
       };
+    case getType(hasApiLevelSupport.success):
+      return {
+        ...state,
+        hasApiLevelSupport: pot.some(action.payload)
+      };
+    case getType(hasApiLevelSupport.failure):
+      return {
+        ...state,
+        hasApiLevelSupport: pot.toError(state.isCieSupported, action.payload)
```
Comment (Undermaken) [inline ts/store/reducers/cie.ts:55]:
```suggestion
        hasApiLevelSupport: pot.toError(state.hasApiLevelSupport, action.payload)
```

File: ts/store/reducers/cie.ts [hunk 6]
```diff
@@ -38,6 +44,26 @@ export default function cieReducer(
...
+      };
+    case getType(hasNFCFeature.success):
+      return {
+        ...state,
+        hasNFCFeature: pot.some(action.payload)
+      };
+    case getType(hasNFCFeature.failure):
+      return {
+        ...state,
+        hasNFCFeature: pot.toError(state.isCieSupported, action.payload)
```
Comment (Undermaken) [inline ts/store/reducers/cie.ts:65]:
```suggestion
        hasNFCFeature: pot.toError(state.hasNFCFeature, action.payload)
```

Comment (CrisTofani) [reply ts/store/reducers/cie.ts]: Fixed in 6533ccb

File: ts/store/reducers/cie.ts (hunk 5 above)
Comment (CrisTofani) [reply ts/store/reducers/cie.ts]: Fixed in 6533ccb

File: ts/screens/authentication/LandingScreen.tsx (hunk 4 above)
Comment (CrisTofani) [reply ts/screens/authentication/LandingScreen.tsx]: Fixed in 6533ccb

File: ts/components/cie/CieNotSupported.tsx (hunk 3 above)
Comment (CrisTofani) [reply ts/components/cie/CieNotSupported.tsx]: Fixed in 6533ccb

**Comment by CrisTofani** [comment]
Aggiungo le gif che mostrano la visualizzazione in condizione del supporto alla CIE

**Pull request merged**